src/lib/pages/backoffice/Acquisition/Order/OrderDetails/OrderDocumentRequest.js

```diff
@@ -0,0 +1,93 @@
+import { PropTypes } from 'prop-types';
+import React from 'react';
+import { withCancel } from '@api/utils';
+import { documentRequestApi } from '@api/documentRequests/documentRequest';
+import { Link } from 'react-router-dom';
+import { BackOfficeRoutes } from '@routes/urls';
+import { Message } from 'semantic-ui-react';
+import { Loader } from '@components/Loader';
+import _isEmpty from 'lodash/isEmpty';
+
+export class OrderDocumentRequest extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cancellableLiterature: {},
```

Comment:
```suggestion
      documentRequest: {},
```
or `record` if you prefer.